srcpkgs/python3-ldap/template

```diff
@@ -1,10 +1,10 @@
 # Template file for 'python3-ldap'
 pkgname=python3-ldap
 _pkgname=python-ldap
```

Comment:
let's remove this variable and replace it with it's value in $distfiles

Reply:
Will do.
Some of the packages have new updates and no longer build or test cleanly so will have to fix that as well.
- apr-util: some cross-compilation issues new in 1.6.3
- virtuoso: test suite fails again in 7.2.9